[Blog] KubeCon EU 2024 talk blog (#5456)
layer5io/layer5: src/collections/blog/2024/03-20-kubeconeu2024/index.mdx

---
title: Layer5 at KubeCon EU 2024
subtitle: Deep Dive into Cloud Native Networking with CNCF TAG Network & Service Mesh WG
date: 2024-03-20

> author: Lee Calcote

Reviewer: ```suggestion
author: Ritik Saxena

```

thumbnail: ./kubeconeu2024.png
darkthumbnail: ./kubeconeu2024.png
description: "Layer5 projects on stage at KubeCon + CloudNativeCon EU 2024"
type: Blog
category: Meshery
tags:
  - KubeCon
  - Meshery

>   - Service Mesh Performance

Reviewer: ```suggestion
  - Service Mesh Performance
  - Nighthawk
```

Explore the extensive service mesh pattern library at [https://meshery.io/catalog](https://meshery.io/catalog) to discover a treasure trove of reusable solutions for your cloud native network.

## Nighthawk Takes Flight: Distributed Performance Analysis

The Service Mesh Performance initiative tackles the challenge of accurately measuring service mesh performance under load. Enter Nighthawk, a distributed performance analysis tool that empowers you to:

* Generate realistic load scenarios that mimic real-world traffic patterns for a more accurate picture of your network's behavior.
* Gain valuable insights into system behavior under pressure, allowing you to make informed decisions about configuration and optimization.
* Compare performance across different service mesh deployments and configurations to identify the best fit for your specific needs.

## Standardization Efforts: MeshMark and the SMP Spec

The CNCF TAG Network actively promotes standardization within the cloud native networking space. Two key initiatives are:

 #### **MeshMark:** 

>  <p>MeshMark serves as a performance index gauging the significance and overhead within your cloud native ecosystem. By transforming performance metrics into actionable insights regarding the significance of individual networking functions within cloud native applications, MeshMark consolidates diverse overhead signals and crucial performance metrics into a straightforward index. </p>

Reviewer: ```suggestion
 <p><a href="https://smp-spec.io/meshmark">MeshMark</a> serves as a performance index gauging the significance and overhead within your cloud native ecosystem. By transforming performance metrics into actionable insights regarding the significance of individual networking functions within cloud native applications, MeshMark consolidates diverse overhead signals and crucial performance metrics into a straightforward index. </p>

```

<img src={MeshMarkFormula}></img>

<p />

 #### **SMP Spec (Service Mesh Performance Specification):**  

>  <p>This initiative establishes a vendor-neutral standard for measuring and reporting service mesh performance. This standardization paves the way for consistent performance comparisons and facilitates informed service mesh selection.</p>

Reviewer: ```suggestion
 <p><a href="https://smp-spec.io">This initiative</a> establishes a vendor-neutral standard for measuring and reporting cloud native infrastructure performance. This standardization paves the way for consistent performance comparisons and facilitates informed infrastructure selection and configuration.</p>

```
